### ISSUE TITLE
About hyperparameters

### ISSUE THREAD

**Issue by DK-Jang** [hiwonjoon/wae-wgan#1]
Hi, wonjoon :)

I want to know values of hyper parameters of MNIST figure in your pretrained model.
I trained the model using default parameters which you wrote down, however the results are something strange and loss value seems to be not converge.

Thank you!

**Comment by hiwonjoon** [comment]
Thanks for your interests.

That's very weird. As far as I remember, the hyperparams I used to generate provided results is same with the written values on the code. I will try to reproduce the result and let you know.

**Comment by hiwonjoon** [comment]
You might want to stick to a W-GAN-based WAE implementation, but please have a look at WAE-MMD implementation, which I updated right now. Since we now avoid tedious, time-consuming training for D, training becomes much faster, and it looks okay with naive hyperparameter I found.

I will close the issue, and please reopen it if you still have the same problem. Cheers! 🍺
